src/Communication/SAF.Communication.PubSub.Cde/SubscriptionRegistry.cs

```diff
@@ -276,6 +273,13 @@ private void HandleSubscriberAlive(TSM message)
                 {
                     _syncSubscribers.ExitWriteLock();
                 }
+                if (string.IsNullOrWhiteSpace(message.PLS))
+                {
+                    var tsmAlive = new TSM(Engines.PubSub, MessageToken.RegistryAlive, _registryIdentity);
+                    tsmAlive.SetToServiceOnly(true);
```

Comment:
When answering to `message.ENG`, we should not send to service only (`SetToServiceOnly(true)`). Remove that line to send the answer TSMs to nodes (e.g. browser instances) too.